great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -357,18 +363,20 @@ def get_domain_records(
 
         if "column_list" in domain_kwargs:
             if "ignore_row_if" in domain_kwargs:
+                column_list = domain_kwargs["column_list"]
```

Comment:
Correct me if I'm wrong but can't we move these two conditionals together in an `and` statement? If `"column_list"` is in `domain_kwargs` and `"ignore_row_if"` isn't, nothing happens right?